modules/util/portal-tools-bundle-support/src/main/resources/META-INF/maven/com.liferay/com.liferay.portal.tools.bundle.support/plugin-help.xml

```diff
@@ -7,7 +7,7 @@
 	<description/>
 	<groupId>com.liferay</groupId>
 	<artifactId>com.liferay.portal.tools.bundle.support</artifactId>
-	<version>3.3.0</version>
+	<version>3.3.1</version>
```

Comment:
The `buildPluginDescriptor` task will automatically update the `plugin-xxx.xml` files.

Please remove those changes or split them out into a separate commit.